index.html

```diff
@@ -29,46 +29,64 @@
             <a href="#contact" class="nav-link">Contact</a>
           </li>
         </ul>
-        <div class="mobile-menu">
-          <span class="bar"></span>
-          <span class="bar"></span>
-          <span class="bar"></span>
-        </div>
-      </nav>
-      <section class="headline">
-        <h1>
-          I’m Drew. <br />
-          Glad to see you!
-        </h1>
-        <p class="content">
-          I’m a software developer! I can help you build a product , feature or
-          website Look through some of my work and experience! If you like what
-          you see and have a project you need coded, don’t hestiate to contact
-          me.
-        </p>
-        <p class="connect">LET'S CONNECT</p>
-        <ul class="social-media">
-          <li>
-            <a href="#"><i class="fa fa-twitter"></i></a>
-          </li>
-          <li>
-            <a href="#"><i class="fa fa-linkedin"></i></a>
-          </li>
-          <li>
-            <a href="#"><i class="fa fa-medium"></i></a>
+        <a href="#" class="mobile-menu" onclick="toggleMobileMenu()">
+          <i class="fa fa-bars"></i>
+        </a>
+        <ul class="mobile-menu-list" id="mobileMenuList">
+          <li class="nav-item">
+            <a href="#portfolio" class="nav-link" onclick="closeMobileMenu()"
+              >Portfolio</a
```

Comment:
- [ ] The mobile menu is not displaying as it should. It shows in a little box window when the hamburger menu is clicked. It actually closes when the X icon is clicked. Kindly take note of a few points when fixing the current issue:
- The mobile should stay fixed and not scroll when the user scrolls up. 👍🏽
- It should be full in width and height of the screen 👍🏽
- The menu items' styles should match the given design 👍🏽

<img width="520" alt="Screenshot 2023-07-25 at 2 59 53 PM" src="https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/50941074/4b7bd3aa-b9a9-4451-bbe4-d9ef6025d26e">